File: C#-OOP-Basics/Problems Course Instance 2018/DefiningClasses/DrawingTool/Square.cs
public class Square : Rectangle
{
    public override int Length
    {
        get
        {
            return this.length;
        }
        set
        {
            this.length = value;
            this.height = value;
        }
    }

    private int length;
    private int height;

    public override int Height
    {
        get
        {
            return this.height;
        }
        set
        {
            this.length = value;
            this.height = value;
        }
    }

    public Square(int side) : base(side, side)
    {
    }
}

File: Java-Advanced/Problems Course Instance 2016_2017/15 - Lambdas and Stream API/examples/edu/pragmatic/Square.java
package edu.pragmatic;

public class Square {

	private int side;
	
	public Square(int side) {
		this.side = side;
	}
	
	public int getSide() {
		return side;
	}
	
	public int calculateArea() {
		return side * side;
	}
	
	@Override
	public String toString() {
		return "Square(" + side + ")";
	}
	
}
